shared/js/participantStore.js

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var loop = loop || {};
+loop.store = loop.store || {};
+
+loop.store.ParticipantStore = (function() {
+  "use strict";
+
+  const PARTICIPANT_SCHEMA = {
+    name: "",
```

Comment:
It works because the property is overwritten. I don't like it so, what if I change the action to:
`UpdatedParticipant: Action.define("updatedParticipant", {
      participantName: String...
}),`

do you agree?